packages/apollo-engine-reporting-protobuf/src/reports.proto

```diff
@@ -202,6 +202,10 @@ message Trace {
 	// instead.
 	string signature = 19;
 
+	// Optional: when GraphQL validation fails, this field can include the entirety of the message
```

Comment:
Does "validation" here mean "parsing or validation"?

Reply:
It was meant to mean "parsing or validation"!